config.js

```diff
@@ -339,9 +339,20 @@ var config = {
     //      "libs/analytics-ga.js", // google-analytics
     //      "https://example.com/my-custom-analytics.js"
     // ],
+    analytics: {
+        // The Google Analytics Tracking ID
+        // The Google Analytics Tracking ID:
+        // googleAnalyticsTrackingId: 'your-tracking-id-UA-123456-1'
 
     // The Google Analytics Tracking ID
     // googleAnalyticsTrackingId = 'your-tracking-id-here-UA-123456-1',
+        // The Amplitude APP Key
+        // The Amplitude APP Key:
+        // amplitudeAPPKey: '<APP_KEY>'
+        // Array with URLs of analytics handler scripts:
+        // analyticsScriptUrls: [ ]
```

Comment:
since we are in the new section, drop the old name

Reply:
Same as above! Will fix it!